tooling/commands/test.js

```diff
@@ -12,6 +12,21 @@ const spawn = require('../util/spawn');
 const {report} = require('../util/coverage');
 const {start, stop} = require('../util/server');
 
+/**
+ * Returns true if the given package should be tested
+ * in the browser. (Some packages are intended for use
+ * only in Node.)
+ * @param {String} packageName
+ * @returns {Boolean}
+ */
+function shouldTestInBrowser(packageName) {
+  const noBrowserPackages = [
+    '@ciscospark/sparkd',
+    '@webex/sparkd'
```

Comment:
Nice! Only thought would be to store the list as a constant or config somewhere.

Reply:
I thought about that, but wanted to reduce the mental overhead of looking elsewhere for the exclusions. If this becomes a large list it makes sense to have it as part of a config.